### DISCUSSION TITLE
Select what fields are displayed?

### DISCUSSION THREAD

**Discussion by MDrooker** [rjsf-team/react-jsonschema-form#2783] [Q&A]
Hi-
I am using a schema that is auto generated from my server based on some DB models.  I dont want to show all the fields, and would like to be able to customize displaying just a few.

In coming from using [Uniform](https://uniforms.tools/), there was an easy way to handle just showing a sub-selection of fields from the schema passed.
We would do something like 
```
 <AutoFields fields={['name']} />
```
to just display the 'name' field

I tried to use the 
```
fields={}
```
property from RJF todo the same thing.  And also tried to just use the uiSchema prop to subselect an object with the fields, but both just continued to show all the fields.

Is there a way to just pass an array of fields to display?

**Comment by MDrooker** [comment]
Any idea how to make the above work with the current code?

**Reply by kfcaio** [reply]
Hi, you can use uiSchema to attribute a specific class name to some fields, let's say `hidden`:

```js
const uiSchema = {
  nameOfOneOfTheFieldsToHide: {
    classNames: "hidden"
  }
};
```

then, use CSS to set `display: none` to all fields with class name `hidden`:

```css
.hidden {
  display: none;
}
```
